src/util/simplify_expr_int.cpp

```diff
@@ -683,6 +683,27 @@ bool simplify_exprt::simplify_bitwise(exprt &expr)
     }
   }
 
+  // now erase 'all ones' out of bitand
+
+  if(expr.id()==ID_bitand)
+  {
+    for(exprt::operandst::iterator
+        it=expr.operands().begin();
+        it!=expr.operands().end();
+        ) // no it++
+    {
+      if(it->is_constant() &&
+         id2string(to_constant_expr(*it).get_value()).find('0')==std::string::npos &&
```

Comment:
The linter complains about this line - you might even want to override the linter, or just break at the `==`?